sdlib/d/gc/global.d

```diff
@@ -46,14 +53,42 @@ private:
 		roots = (cast(const(void*)[]*) ptr)[0 .. length];
 
 		import d.gc.range;
-		roots[index] = makeRange(range);
+		if (range.length == 0) {
```

Comment:
This trick needs to be documented somewhere int he comments.

Reply:
ok